src/core/evaluator.js

```diff
@@ -2058,23 +2071,28 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         coded: false
       };
 
-      if (composite) {
-        var cidEncoding = baseDict.get('Encoding');
-        if (isName(cidEncoding)) {
-          properties.cidEncoding = cidEncoding.name;
-        }
-        properties.cMap = CMapFactory.create(cidEncoding,
-          { url: PDFJS.cMapUrl, packed: PDFJS.cMapPacked }, null);
-        properties.vertical = properties.cMap.vertical;
-      }
-      this.extractDataStructures(dict, baseDict, xref, properties);
-      this.extractWidths(dict, xref, descriptor, properties);
-
-      if (type === 'Type3') {
-        properties.isType3Font = true;
-      }
+      return this.extractDataStructures(dict, baseDict, xref, properties)
+      .then(function (properties) {
+        this.extractWidths(dict, xref, descriptor, properties);
 
-      return new Font(fontName.name, fontFile, properties);
+        if (type === 'Type3') {
+          properties.isType3Font = true;
+        }
+        if (composite) {
+          var cidEncoding = baseDict.get('Encoding');
+          if (isName(cidEncoding)) {
+            properties.cidEncoding = cidEncoding.name;
+          }
+          return CMapFactory.create(cidEncoding,
```

Comment:
Order of CMapFactory.create and extractDataStructures was changed, it needs to be reversed